draft-ietf-rtcweb-jsep.xml

```diff
@@ -576,6 +576,135 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
+            </list>
+            Thus, a system with a HD decoder might send</t>
+
+            <t><list style="symbols">
+              <t>a=imageattr:* recv [x=[16:1920],y=[16:1080],q=0.1]</t>
+              </list>If the Javascript has constrained the stream to have
+            width={min:320, max:640}, the result might be</t>
+
+            <t><list style="symbols">
+              <t>a=imageattr:* recv [x=[16:1920],y=[16:1080],q=0.1]
+              [x=[320:640],y=[16:1080],q=0.5]</t>
+              </list>and if the stream is known to be displayed on a surface of
+            dimension 640x480, the result might be</t>
+
+            <t><list style="symbols">
+              <t>a=imageattr:* recv [x=[16:1920],y=[16:1080],q=0.1]
+              [x=[320:640],y=[16:1080],q=0.5] [x=640,y=480,q=1.0]</t>
+            </list>
+            The "par=" parameter is used to represent a constraint on
+            the aspect ratio. The numbers given are produced from the
+            user-specified value by rounding the minimum down to four decimal
+            digits, and rounding the maximum up to four decimal digits.</t>
+          </section>
+
+          <section title="Choice of imageattr send attributes">
+            <t>There are two cases for generation of the send
+            attributes: When the correspondent's recv attribute is
+            unknown, and when the recv attribute is known.
+            </t>
+            <t>When the recv attribute is unknown, such as for an
+            initial offer, the send attribute is generated as follows:
+            </t>
+            <t><list style="symbols">
+              <t>q=0.1 is used to indicate the range of video sizes the
+              system is capable of generating.</t>
+              <t>q=0.5 is used to indicate the range of video sizes
+              that are allowed within present constraints for the
+              attached video stream. If no video stream is attached,
+              this is omitted.</t>
+              <t>q=1.0 is used for the presently generated video
+              format.</t>
+            </list></t>
+            <t>When the recv attribute is known, and a video stream is
+            attached, the sender MAY choose to act as if the set of
+            formats given were a set of constraints, and apply those to the
+            video stream, in addition to any user-specified
+            constraints,
+            before generating the send attribute. Note
+            that in the case of non-WebRTC-generated "recv"
+            attributes, the lower-priority configurations might not be
+            subsets of the higher-priority ones, so they need to be
+            applied as "advanced" constraints, in order from highest
+            to lowest priority, in order to give a reasonable result
+            for all cases, and the application of some constraints
+            will likely fail.
+            </t>
+            <t>This will generate an "a=imageattr send" attribute
```

Comment:
I'm not sure I understand the purpose of a send attribute. The recv is pretty clear, but what good does it do to say what you are sending?